js/data/bucket.js

```diff
@@ -282,7 +282,7 @@ Bucket.create = function(options) {
     let type = options.layer.type;
     if (type === 'fill' && (!options.layer.isPaintValueFeatureConstant('fill-extrude-height') ||
         !options.layer.isPaintValueZoomConstant('fill-extrude-height') ||
-        options.layer.getPaintValue('fill-extrude-height') !== 0)) {
+        options.layer.getPaintValue('fill-extrude-height', {zoom: this.zoom}) !== 0)) {
```

Comment:
Oops, good catch @jfirebaugh. Following up in https://github.com/mapbox/mapbox-gl-js/commit/79dd26a2cebfb9a6b0543aa2bdf8672cd179339a